apps/tasks/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.9 on 2021-11-14 10:31
+
+from django.db import migrations, models
+import django.db.models.deletion
+import enumchoicefield.fields
+import tasks.models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Task',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False,
+                 primary_key=True, serialize=False, unique=True)),
+                ('title', models.CharField(max_length=200)),
+                ('created_date', models.DateTimeField(auto_now_add=True)),
+                ('priority', enumchoicefield.fields.EnumChoiceField(
+                    default=tasks.models.Priority(1), enum_class=tasks.models.Priority, max_length=1)),
```

Comment:
maybe you can write it like:
```suggestion
                    default=tasks.models.Priority.MEDIUM, enum_class=tasks.models.Priority, max_length=1)),
```
to me, it looks more readable like that

Reply:
Don't you think it should be low by default?